aiohttp/client.py

```diff
@@ -225,36 +225,34 @@ class ClientTimeout:
 class ClientSession:
     """First-class interface for making HTTP requests."""
 
-    ATTRS = frozenset(
-        [
-            "_base_url",
-            "_source_traceback",
-            "_connector",
-            "requote_redirect_url",
-            "_loop",
-            "_cookie_jar",
-            "_connector_owner",
-            "_default_auth",
-            "_version",
-            "_json_serialize",
-            "_requote_redirect_url",
-            "_timeout",
-            "_raise_for_status",
-            "_auto_decompress",
-            "_trust_env",
-            "_default_headers",
-            "_skip_auto_headers",
-            "_request_class",
-            "_response_class",
-            "_ws_response_class",
-            "_trace_configs",
-            "_read_bufsize",
-            "_max_line_size",
-            "_max_field_size",
-            "_resolve_charset",
-            "_default_proxy",
-            "_default_proxy_auth",
-        ]
```

Comment:
Yeah, this needs to be kept because it differs on this branch. The patch needs to be adapted for this branch to make an equivalent single-line change in diff.

Reply:
Ok